src/main/kotlin/com/exactpro/th2/check1/rule/AbstractCheckTask.kt

```diff
@@ -608,6 +610,18 @@ abstract class AbstractCheckTask(
             null
         }
 
+    private fun TaskTimeout.getOrCalculateMessageTimeout(): Long {
+        if (this.timeout > 0L) {
+            return timeout
+        }
+        require(messageTimeout != null && messageTimeout > 0) {
+            "Timeout cannot be calculated because 'timeout' and 'message timeout' is not set"
+        }
+        return messageTimeout.also {
+            LOGGER.info("Rule `timeout` is not specified, used `message timeout` instead")
+        }
```

Comment:
Don't see any reason to use `also` here. You can easily split this
```suggestion
        LOGGER.info("Rule `timeout` is not specified, used `message timeout` instead")
        return messageTimeout
```